mr/pipeline.cc

```diff
@@ -92,6 +95,17 @@ bool Pipeline::Run(Runner* runner) {
     ProcessTable(sptr.get());
   }
 
+  VLOG(1) << "Saving counter maps";
+  for (const auto& name_and_map : metric_maps_) {
+    std::string to_write;
+    for (const auto& k_v : name_and_map.second) {
+      std::string escaped_first = absl::StrReplaceAll(k_v.first, {{"\"", "\"\""}});
```

Comment:
I think we can assume that counter names do not have double brackets.

Reply:
so no need for adding brackets and escaping, imho

Reply:
Better safe than sorry, don't you think?
Vorsicht ist besser als Nachsicht, denkst du nicht?